ci/requirements-3.5.sh

```diff
@@ -8,4 +8,4 @@ echo "install 35"
 conda remove -n pandas python-dateutil --force
 pip install python-dateutil
 
```

Comment:
can you check if we have pyarrow installed in ci/* anywhere else (and if so update the pinned versions)

Reply:
I checked and changed any pinned versions to the new minimum. If it wasn't already pinned I left it as is since a normal install will pull in later versions that the minimum.

Reply:
![image](https://user-images.githubusercontent.com/881019/33705822-888b68dc-db7d-11e7-86db-f429e370406c.png)